positioning1.css

```diff
@@ -0,0 +1,27 @@
+div{
+    width: 250px;
+    height: 250px;
+}    
+
+.yellow{
+    background-color: yellow;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+
+}
+
+.blue{
+    background-color: blue;
+    position: absolute;
+    top: 125px;
```

Comment:
Poti si aici folosi 50%, ca mai sus. 
Pentru ca, acest copil este in interiorul lui yellow, care are aceasi dimensiune, o sa se alinieze fix la mijloc